api/shared/labels.go

```diff
@@ -32,4 +32,10 @@ const (
 	// OpenStackNetConfigReconcileLabel - label set on objects on which change
 	// trigger a reconcile of the osnetconfig
 	OpenStackNetConfigReconcileLabel = "osnetconfig-ref"
+	// OpenStackBaremetalSetAppLabel -
+	OpenStackBaremetalSetAppLabel = "osp-baremetalset"
+	// OwnerControllerNameLabelSelector -
+	OwnerControllerNameLabelSelector string = "osp-director.openstack.org/controller"
+	// OwnerUIDLabelSelector -
+	OwnerUIDLabelSelector string = "osp-director.openstack.org/uid"
```

Comment:
I know this is duplicating what we have in the main `common` package.  Moving all the constants to the API just seems like a lot of work for this initial pass.